core/services/ocr2/plugins/ocr2keeper/evm21/registry.go

```diff
@@ -38,18 +38,21 @@ import (
 )
 
 const (
-	// DefaultAllowListExpiration decides how long an upkeep's allow list info will be valid for.
-	DefaultAllowListExpiration = 20 * time.Minute
-	// CleanupInterval decides when the expired items in cache will be deleted.
-	CleanupInterval = 25 * time.Minute
+	// defaultAllowListExpiration decides how long an upkeep's allow list info will be valid for.
+	defaultAllowListExpiration = 20 * time.Minute
+	// cleanupInterval decides when the expired items in cache will be deleted.
+	cleanupInterval = 25 * time.Minute
+	// LookbackDepth decides valid trigger block lookback range
+	LookbackDepth = 1024
+	// BlockHistorySize decides the block history size
+	BlockHistorySize = 128
```

Comment:
I think these make more sense to be defined as constants within block_subscriber.go instead of passing as arguments from here

Reply:
done